common/src/chain/config/mod.rs

```diff
@@ -587,30 +588,74 @@ fn create_testnet_genesis() -> Genesis {
     )
 }
 
-pub fn regtest_genesis_values() -> (
+#[derive(Clone, Debug)]
+pub struct GenesisStakingSettings(String);
+
+impl GenesisStakingSettings {
+    pub fn new(settings: String) -> Self {
+        Self(settings)
+    }
```

Comment:
If you parse the string in the constructor, you could take a `&str` rather than `String` to initialize the struct.

Reply:
I keep forgetting this. Fixed.